build/build.sh

```diff
@@ -92,6 +94,10 @@ while [[ $# -gt 0 ]]; do
     --test)
       test=true
       shift 1
+      ;;
+	--Perf)
+      pack=true
```

Comment:
Shouldn't this be `perf=true`?  Looks like a copy/paste error.

Reply:
Yup, done...